src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.test.mocks.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+jest.doMock('lodash', () => {
+  const original = jest.requireActual('lodash');
+  return {
+    ...original,
+    get: (func: Function) => {
+      function get(this: any, args: any[]) {
+        return func.apply(this, args);
+      }
+      return get;
+    },
```

Comment:
Where is this used?

Reply:
We use lodash `get` in `canViewInApp` where we check to see if the object has the right [capabilities enabled](https://github.com/elastic/kibana/pull/112034/files/8c923173d37711cbe40e87e083dd3672cfd4704b#diff-20257a35467392454961fa1044dfcc249dd944d776ff709c73fd3435c8dce2b4R90).